modules/Books.js

```diff
@@ -0,0 +1,53 @@
+const getBooksag = JSON.parse(localStorage.getItem('books'));
+const booksData = [];
+
+class BooksClass {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  add() {
+    const title = document.querySelector('#title').value;
+    const author = document.querySelector('#author').value;
+    const book = {
+      Title: title,
+      author,
+    };
+
+    const netBooks = JSON.parse(localStorage.getItem('books'));
+    if (!netBooks) {
+      booksData.push(book);
+      localStorage.setItem('books', JSON.stringify(booksData));
+    } else {
+      netBooks.push(book);
+      localStorage.setItem('books', JSON.stringify(netBooks));
+    }
+    window.location.reload();
+  }
+
+  showBooks() {
+    const divP = document.querySelector('#booksIn');
+    const getBooks = JSON.parse(localStorage.getItem('books'));
+
+    if (!getBooks || !getBooks[0]) {
+      divP.innerHTML = '<div class="no-cards">No Books to Show...</div>';
+    } else {
+      for (let i = 0; i < getBooks.length; i += 1) {
+        const temp = i;
+        divP.innerHTML += `
+        <div class="cards" data-id="${temp}">
+            <p>"${getBooks[i].Title}" by ${getBooks[i].author}</p><button class="remove" data-id='${temp}'>Remove</button>
+        </div>
+        `;
+      }
+    }
+  }
+
+  del(id) {
+    getBooksag.splice(id, 1);
+    localStorage.setItem('books', JSON.stringify(getBooksag));
+  }
+}
+
+export default BooksClass;
```

Comment:
- [ ] Good Job so far your project is awesome and everything works as it should, however, as part of the requirements we should stick with ES6 syntax, kindly update all of your BooksClass functions to an arrow functions example : 

```JS
  // Normal Function
  del(id) {
     ...
  }
  // Arrow Function ES6
   del = (id) => {
     ...
  }

```